site/developer-tools.html

```diff
@@ -505,8 +505,8 @@ <h4>Solving a binary incompatibility</h4>
 
 <p>For the problem described above, we might add the following:</p>
 
-<figure class="highlight"><pre><code class="language-scala" data-lang="scala"><span class="c1">// [SPARK-zz][CORE] Fix an issue
-</span><span class="nc">ProblemFilters</span><span class="o">.</span><span class="n">exclude</span><span class="o">[</span><span class="kt">DirectMissingMethodProblem</span><span class="o">](</span><span class="s">"org.apache.spark.SomeClass.this"</span><span class="o">)</span></code></pre></figure>
+<figure class="highlight"><pre><code class="language-scala" data-lang="scala"><span class="c1">// [SPARK-zz][CORE] Fix an issue</span>
+<span class="nv">ProblemFilters</span><span class="o">.</span><span class="py">exclude</span><span class="o">[</span><span class="kt">DirectMissingMethodProblem</span><span class="o">](</span><span class="s">"org.apache.spark.SomeClass.this"</span><span class="o">)</span></code></pre></figure>
```

Comment:
why `nc` -> `nv`?

Reply:
btw, do we have `class="py"`? I don't see `py` in the css in current site, or I missed it.

Reply:
@viirya those changes in the second commit(3b416ecb027c9262a28ad778d50634d6bd582f81) were made by running `jekyll build`  in dir `spark-website`.

Reply:
oh ok. then it should be fine, i don't see change in developer-tools related to this, maybe related to jekyll.

Reply:
I tried it locally. Just run `jekyll build` can reproduce these changes.

Reply:
Thank you for confirming